lib/Doctrine/Migrations/Provider/OrmSchemaProvider.php

```diff
@@ -10,6 +10,9 @@
 use UnexpectedValueException;
 use function count;
 
+/**
+ * @internal
```

Comment:
should not be internal. ORM users should be able to use this implementation of the non-internal `SchemaProviderInterface`

Reply:
This gets used internally here. I don't think it should be public https://github.com/doctrine/migrations/blob/master/lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php#L136

Reply:
If they don't provide their own schema provider that implements the `SchemaProviderInterface` then it uses the OrmSchemaProvider.